test/component_test.go

```
@@ -0,0 +1,218 @@
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+	}
+
+	defer clientset.NetworkingV1().Ingresses(ingressNamespace).Delete(context.Background(), ingressName, metav1.DeleteOptions{})
```

golangci-lint (1.64.8)

Error return value of `(k8s.io/client-go/kubernetes/typed/networking/v1.IngressInterface).Delete` is not checked

(errcheck)